packages/lexical-editor/src/components/ToolbarActions/QuoteAction.tsx

```diff
@@ -14,13 +14,11 @@ export const QuoteAction = () => {
             // Try to set default quote style, when the action button is clicked for first time
             const DEFAULT_QUOTE_ID = "quote";
             const hasQuoteStyles = themeEmotionMap && themeEmotionMap[DEFAULT_QUOTE_ID];
-
             formatToQuote(editor, hasQuoteStyles ? DEFAULT_QUOTE_ID : undefined);
-            setIsActive(true);
             return;
         }
+        debugger;
```

Comment:
```suggestion

```

Reply:
> A couple of comments, but I'd also like to report something I noticed ⬇️
> 
> The video shows me selecting a block that was created previously, with an older version of Lexical. Clicking on it throws an `parseEditorState: type "base-heading-node" + not found` error.
> 
>  old-block.mov 
> Not sure if we want to tackle this in this PR or not. Prolly not as it's related to header. But it is something we should double check.

You are using the old version of the heading node before we did the refactoring, just create a new block with the new heading element and everything will be ok, in the future we will not change this node type name

Will not be problem for users because they don't have this heading node.

Reply:
debugger is removed, thanks